src/app/item-page/edit-item-page/item-status/item-status.component.scss

```diff
@@ -0,0 +1,3 @@
+.status-label {
+  font-weight: 700;
+}
```

Comment:
I would add [the css class `font-weight-bold`](https://getbootstrap.com/docs/4.6/utilities/text/#font-weight-and-italics) to the HTML rather than setting it in css. 

The reason for that is: if you want to change _how_ bold the text is in your theme. Then you can do it in a single place if we use the same css class everywhere. If we don't, you'd have to find all instances of font-weight in scss files across the app to change it

Reply:
Good morning @artlowel!

I've already adjusted the code. In the “item-status.component.html” the labels are defined directly in the divs, if it's interesting for you, I could add the <span> tag as it is defined in the “item-operation.component.html”.

I hope I've got it right and made the necessary adjustments to the code. 

Once again, we are at your disposal!